README.md

```diff
@@ -251,6 +251,9 @@ Image accepts the following env variables:
 - `HIDDEN_OPTIONS_PATH` (alternative to `HIDDEN_OPTIONS`): useful if attaching a volume containing a `hiddenOptions.yaml` file definining which languages to hide. e.g. `/data/hiddenOptions.yaml`
 - `HIDDEN_OPTIONS` (alternative to `HIDDEN_OPTIONS_PATH`): allows to pass hidden options as an env variable, in the format `{category}:{language},{language},{language}|{category}:{language},{language},{language}`
 e.g. `servers:foo,bar|clientsV3:wtf,isthis` where category can be `clients`, `servers`, `clientsV3`, `serversV3`
+- `AVAILABLE_OPTIONS_PATH` (alternative to `AVAILABLE_OPTIONS`): useful if attaching a volume containing a `availableOptions.yaml` file definining which languages to hide. e.g. `/data/availableOptions.yaml`
```

Comment:
`which languages to hide` should be `which languages to expose` or something I guess